samples/ask-akka-agent/src/main/resources/application.conf

```diff
@@ -1,7 +1,7 @@
 akka.javasdk {
   agent {
-    # Other AI models can be configured, see https://doc.akka.io/java/agents.html#model
-    # and https://doc.akka.io/java/model-provider-details.html for the reference configurations.
+    # Other AI models can be configured, see https://doc.akka.io/sdk/agents.html#model
+    # and https://doc.akka.io/sdk/model-provider-details.html for the reference configurations.
```

Comment:
this should also be changed in the samples that are maintained in https://github.com/akka-samples

Perhaps too tedious and we can rely on the redirect, but at least update https://github.com/akka-samples/empty